serverlessworkflow/modules/ROOT/pages/cloud/common/_proc_deploy_sw_quarkus_cli.adoc

```diff
@@ -0,0 +1,84 @@
+// links
+:quarkus_k8s_deploy_url: https://github.com/quarkusio/quarkus/issues/26385
+
+[[proc-deploy-sw-application-quarkus-cli]]
+=== Deploying your workflow application using Quarkus CLI
+
+.Prerequisites
+* Workflow application is installed.
+* Quarkus CLI is installed. +
+For more information, see link:{quarkus_cli_url}[Building Quarkus Apps with Quarkus command line interface (CLI)].
+* You have completed the steps in link:#proc-deploy-sw-application-kubectl[Deploying your workflow application using `kubectl`]
```

Comment:
Why do you need to complete the previous section ? You are doing the same here but with the quarkus CLI

Reply:
good point, there is no need, removed.